assets/js/theme/global/quick-view.js

```diff
@@ -22,6 +22,8 @@ export default function (context) {
 
             modal.$content.find('[data-slick]').slick();
 
+            modal.setupFocusableElements('forQuickView');
```

Comment:
Use modalType map value instead of srtring

Reply:
Do you mean to create mapping object in modal.js and using it in quick-view.js after creating a modal?

Reply:
It's about map with modal types constants (example ModalTypes.QUICK_VIEW)